manager.py

```diff
@@ -143,8 +143,11 @@ def get_service_logger_names(self, service_name, add_level):
                                     add_level=add_level)
         return self._service_manager.execute_request(service_name, request)
 
-    @staticmethod
-    def get_log_entries(name, entries_count=-1, level=None, component=None):
+    def _service_list(self):
+        return self._service_manager.instances.configuration.get_children()
```

Comment:
@f1401martin do you know a more obvious way to get the list of service names?

Reply:
maybe, not that this is wrong, but I think self._service_manager.get_process(service_name) could be used instead, which is what "ServiceManager.execute_request" ends up using, and then the "_service_list" new method would not be that relevant, should be more efficient too!

Reply:
Discard previous comment, get_process only works for running services, we could use this intead
```python
        if name:
            for item in self._service_manager.instance_properties:
                if item.config.name == name:
                    filename = path.join(
                        NIOEnvironment.get_path("logs"), "{}.log".format(name)
                    )
                    if not path.isfile(filename):
                        return []
                    return LogEntries.read(filename, entries_count,
                                           level, component)
            else:
                raise ValueError("{} service does not exist".format(name))
```